src/assets/js/components/Installments/InstallmentsTotal/InstallmentsTotal.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import {IntlProvider} from "react-intl";
+import { InstallmentsTotal } from "./InstallmentsTotal";
+import {render, screen} from '@testing-library/react'
+describe('InstallmentsTotal', () => {
+  it('should display the right total amount', () => {
+    render(<IntlProvider locale={"fr"}><InstallmentsTotal totalAmount={1000}/></IntlProvider>)
```

Comment:
there is a warning about missing translations in the test's console that we need to check